api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
I believe this should not be removed.  Which Go version are you using?

Reply:
16

i go fix